hardware/1Wire/1WireByKernel.cpp

```diff
@@ -23,9 +23,10 @@
 #endif //_DEBUG
 
 
-C1WireByKernel::C1WireByKernel() :
-   m_AllDevicesInitialized(false)
+C1WireByKernel::C1WireByKernel()
 {
+   ThreadBuildDevicesList();
```

Comment:
This might take a long time and causes domoticz to start slow when you have a lot of sensors
That is probably why this is done in the ThreadFunction
You could perform this in StartHardware